pkg/trace/config/config.go

```diff
@@ -428,6 +428,9 @@ type AgentConfig struct {
 	// RemoteSamplingClient retrieves sampling updates from the remote config backend
 	RemoteSamplingClient RemoteClient `json:"-"`
 
+	// RemoteCustomTagsClient retrieves custom tags updates from the remote config backend
+	RemoteCustomTagsClient RemoteClient `json:"-"`
```

Comment:
I believe `RemoteClient` is an interface around a remote config client and in general we want only one of these to exist in an app at any given time.

I think the variable name here is RemoteSamplingClient but I think it's actually already handling APM_SAMPLING and AGENT_CONFIG RC products, so it's actually a more generic client than its name.

Reply:
Similarly, you can delete this now that we've discussed how to use the existing `RemoteConfigClient`